.cirrus.yml

```diff
@@ -1,5 +1,10 @@
+env:
+  GH_API_TOKEN: ENCRYPTED[18bd1bb47022f0ec649d768dd4b822a44ed5e0e378ea7f8703b2a55cd6535ab90a50e915d09fe48eae451bdb9b63aeae]
```

Comment:
@wfurt, please follow these steps to get your encrypted token for publishing to GH releases: https://github.com/wfurt/freebsd-bootstrap-cli/pull/2#discussion_r373858270

Reply:
I did generate token while back but I did not find way how to turn into encrypted form ;(